src/app/lib/layoutGrid.js

```diff
@@ -163,3 +175,57 @@ export const layoutGridItemSmall = css`
 export const layoutGridItem = css`
   grid-column: 1 / -1;
 `;
+
+export const captionGridItemLandscapeSquare = css`
```

Comment:
in the original definition of the grid columns square and landscape appeared to be the same size. however this needs to change anyway because Natalie has corrected us with this:

> Group 5: landscape - 5-16/20.
> Group 5: portrait - 5-12/20.
> Group 5: square - 5-14/20.